src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/SignCommand.cs

```diff
@@ -232,6 +232,17 @@ private static StoreName ValidateAndParseStoreName(CommandOption store)
                 // Thow if the user provided a fingerprint and a subject
                 throw new ArgumentException(Strings.SignCommandMultipleCertificateException);
             }
+            else if (fingerprint.Value() != null)
```

Comment:
What used to happen if the user specified an invalid fingerprint value?  Just curious.

Reply:
Previously, if the certificate fingerprint was invalid (e.g., a string with non-hex characters or an empty string), it would still pass to the Core libraries and fail later while searching for certificates. With this PR, we are failing fast if the certificate fingerprint is invalid.